proto/regen/ecocredit/v1/tx.proto

```diff
@@ -462,11 +462,8 @@ message MsgBridge {
   // recipient is the address of the account receiving the bridged credits.
   string recipient = 3;
 
-  // recipient is the address of the contract handling the bridged credits.
-  string contract = 4;
```

Comment:
The logic for this was already removed as we are now using batch contract mapping to check if credits can be bridged back to the source chain. If a batch contract mapping exists, the stored contract is used. It looks like I forgot to remove this field or I accidentally added it back in with a merge.